redisson/src/main/java/org/redisson/RedissonMap.java

```diff
@@ -66,7 +66,7 @@
  */
 public class RedissonMap<K, V> extends RedissonExpirable implements RMap<K, V> {
 
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final Logger log = LoggerFactory.getLogger(RedissonMap.class);
```

Comment:
this needs to remain unchanged or be added to all inheritances with concrete class, not RedissonMap.